charts/intel-gpu-resource-driver/templates/nfd.yaml

```diff
@@ -90,9 +80,7 @@ spec:
             value:
             - "8086"
     name: intel.gpu.generic.count.380
-  - labels:
-      gpu.intel.com/product: "Max_1100"
-    labelsTemplate: "gpu.intel.com/device.count={{"{{"}} len .pci.device {{"}}"}}"
+  - labelsTemplate: "gpu.intel.com/device.count={{"{{"}} len .pci.device {{"}}"}}"
```

Comment:
There are 4 rules for different sets of GPU IDs, each producing the same `gpu.intel.com/device.count=<count>` label.  Which won't produce correct results on node which has GPUs from multiple rule sets.

While this is not a problem for GPU plugin because it does not support nodes with different types of Intel GPUs, DRA GPU driver does support such nodes.

I think it's best just to remove these rules completely for now...

Reply:
thanks! I removed these rules